ArcGISRuntimeSDKQt_CppSamples/Layers/TileCacheLayer/Info.plist

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDisplayName</key>
+	<string>TileCacheLayer</string>
```

Comment:
The naming rule is that we use the name defined in the readme design. In this case, `ArcGIS tiled layer (tile cache)`. This name change should be applied throughout.